examples/Basic/Main.elm

```diff
@@ -43,8 +43,26 @@ userDefinedDatePickerSettings today =
             defaultSettings UpdatePicker
     in
     { defaults
-        | dayDisabled = \datetime -> isDateBeforeToday (Time.floor Day Time.utc today) datetime
+        | dateTimeProcessor =
+            { isDayDisabled = \datetime -> isDateBeforeToday (Time.floor Day Time.utc today) datetime
+            , allowedTimesOfDay =
+                \datetime ->
+                    let
+                        processingParts =
+                            Time.posixToParts Time.utc datetime
+
+                        todayParts =
+                            Time.posixToParts Time.utc today
+                    in
+                    if processingParts.day == todayParts.day && processingParts.month == todayParts.month && processingParts.year == todayParts.year then
+                        { startHour = todayParts.hour, startMinute = todayParts.minute, endHour = 16, endMinute = 30 }
+
+                    else
+                        { startHour = 8, startMinute = 0, endHour = 16, endMinute = 30 }
+            }
         , today = Just today
```

Comment:
I got a bit confused on this `today` setting option and thought it is actually necessary for the datepicker to work correctly - but it's only used to allow a special styling for the current day. Maybe renaming the `today` option to something like `focusedDay` might be less confusing. Also it would fit better, if someone wants to use this option for another purpose